mlrun/api/schemas/feature_store.py

```diff
@@ -7,9 +7,13 @@
 class Feature(BaseModel):
     name: str
     value_type: str
+    labels: Optional[dict]
 
-    class Config:
-        orm_mode = True
+
+class Entity(BaseModel):
+    name: str
+    value_type: str
+    labels: Optional[dict]
```

Comment:
I wonder whether we should make `Entity` and `Feature` to also have `metadata` and `spec` (currently no `status` I guess) WDYT ?
@yaronha ?

Reply:
@Hedingber no need, they are under featureset